src/main/java/online/talkandtravel/controller/websocket/UserEventController.java

```diff
@@ -0,0 +1,22 @@
+package online.talkandtravel.controller.websocket;
+
+import java.security.Principal;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import online.talkandtravel.service.OnlineService;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@Log4j2
+public class UserEventController {
```

Comment:
i think this controller is redundant and suggest to move this method into EventController